sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/FilesImpl.java

```diff
@@ -108,7 +108,7 @@ public FilesImpl(AzureFileStorageImpl client) {
         @Put("{shareName}/{filePath}")
         @ExpectedResponses({201})
         @UnexpectedResponseExceptionType(FileStorageException.class)
```

Comment:
Are we planning to change the name to `ShareFileStorageException` or `ShareStorageException`?

Reply:
ShareStorageException now